app/routes.py

```diff
@@ -0,0 +1,444 @@
+    # video = Video.query.get(video_id) 
+    # video.title = request_body["title"]
+    # video.release_date = request_body["release_date"]
+    # video.total_inventory = request_body["total_inventory"]
+    # db.session.commit()
+    # return make_response(video.to_dict(), 200)
+
+@video_bp.route("/<video_id>", methods=["DELETE"])
+def delete_video(video_id):
+    check = check_customer_video_data(method="DELETE", 
+                                        id=video_id, 
+                                        model=Video,
+                                        entity="Video")
+    if check:
+        return check
+    video = Video.query.get(video_id) 
+    db.session.delete(video)
+    db.session.commit()
+    response = {"id": video.id}
+    return make_response(response, 200)
+
+@video_bp.route("/<video_id>/rentals", methods=["GET"])
+def read_rentals_by_video(video_id):
+    check = check_customer_video_rental_data(id=video_id, 
+                                                model=Video, 
+                                                entity="Video", 
+                                                method="GET")
+    if check:
+        return check
+    rentals = db.session.query(Rental).filter(Rental.video_id==video_id)
+    customers=[Customer.query.get(rental.customer_id) for rental in rentals]
+    rental_response = [customer.to_dict() for customer in customers]
+    return jsonify(rental_response), 200
+
+#----------------------------------------------------------------------------------#
+#---------------------------   Rental Endpoints     -------------------------------#
+#----------------------------------------------------------------------------------#
+rental_keys = ["customer_id", "video_id"]
+
+@rental_bp.route("/check-out", methods=["POST"])
+def handle_check_out():
+    request_body = request.get_json()
+    check = check_customer_video_rental_data(request_body=request_body, method="POST")
+    if check:
+        return check
+    else: 
+        video_id = request_body["video_id"]
+        video = Video.query.get(video_id)
+        if calculate_inventory_available(video):
+            return process_checkout(request_body, video)
+        else:
+            return make_response({"message": "Could not perform checkout"}, 400)
+
+@rental_bp.route("/check-in", methods=["POST"])
+def handle_check_in():
+    request_body = request.get_json() 
+    check = check_customer_video_rental_data(request_body=request_body, method="POST")
+    if check:
+        return check
+    else:
+        return process_checkin(request_body)
+
+
+#----------------------------------------------------------------------------------#
+#---------------------------    Helper Functions    -------------------------------#
+#----------------------------------------------------------------------------------#
+
+#####---------------------    Model Object Creation   -------------------------#####
+def create_customer(request_body):
```

Comment:
I recommend renaming this, "create_customer_helper" for example.  Using the same name creates some readability issues, and can also cause weird behavior issues in Python.